### DISCUSSION TITLE
Can not flash Technic Hub, takes too long to respond (halfway through)

### DISCUSSION THREAD

**Discussion by Devilly** [pybricks/support#1360] [Q&A]
I'm trying to flash my Technic Hub with the latest Pybricks version. Starting the process works but at some point it always reaches this error:

![image](https://github.com/pybricks/support/assets/603645/e9af1d9e-c9bb-4818-84da-e361f86f65bc)

I had an older Pybricks version flashed on the hub, which at the very least means the process was able to finish properly before. I'm not sure what to do now. Starting the hub in the normal way now also causes the flashing purple lights.

**Comment by dlech** [comment]
Windows Bluetooth is particularly problematic. If you have another device, like an Android phone, you can use that to flash the firmware. 

More info at #270. If you want to dig deeper and log Bluetooth packets as described there, that could help us get to the bottom of the problem.
